src/web/modules/payouts/payouts_show.njk

```diff
@@ -0,0 +1,60 @@
+{% extends "layout/layout.njk" %}
+
+{% block main %}
+  <span class="govuk-caption-m">{{service.name}}</span>
+  <h1 class="govuk-heading-m">Stripe payouts</h1>
+
+  <div>
+    <a href="/services/{{ service.external_id }}" class="govuk-back-link">Service ({{ service.name }})</a>
+  </div>
+
+  <table class="govuk-table">
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header" scope="col">Stripe ID</th>
+        <th class="govuk-table__header" scope="col">Amount</th>
+        <th class="govuk-table__header" scope="col">Status</th>
+        <th class="govuk-table__header" scope="col">Initiated</th>
+        <th class="govuk-table__header" scope="col">Est. Arrival</th>
+      </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+      {% for payout in payouts.data %}
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell" title="{{payout.id}}}">
+          <a class="govuk-link" href="payouts/{{payout.id}}" target="_blank">{{payout.id | truncate(10)}}</a>
+        </td>
+        <td class="govuk-table__cell govuk-table__cell--numeric">{{payout.amount | currency}}</td>
+        <td class="govuk-table__cell"><strong class="govuk-tag">{{payout.status}}</strong></td>
+        <td class="govuk-table__cell">{{payout.created | unixDate }}</td>
+        <td class="govuk-table__cell">{{payout.arrival_date | unixDate }}</td>
+      </tr>
+      {% else %}
+      <tr class="govuk-table__row">
+        <td colspan="5" class="govuk-table__cell">No payouts found for {{service.name}}</td>
+      </tr>
+      {% endfor %}
+    </tbody>
+  </table>
+
+  <div class="toolbox-right">
+    <a
+    {% if startingAfter or (payouts.has_more and endingBefore) %}
+      href="?endingBefore={{ payouts.data[0].id }}"
```

Comment:
what will happen if there aren't any payouts (new service)?

Reply:
`has_more` from Stripe will only be sent in the response if you have a full page (reached the `limit`). Neither of these buttons will be enabled until `has_more` has been sent as true, if there aren't any payouts both Next and Previous will be disabled